client/package.json

```diff
@@ -19,36 +19,44 @@
     "@babel/polyfill": "^7.0.0",
     "@babel/preset-env": "^7.1.5",
     "@babel/preset-react": "^7.0.0",
+    "@epegzz/sass-vars-loader": "^4.0.0",
     "autoprefixer": "^9.3.1",
     "babel-loader": "^8.0.4",
     "classnames": "^2.2.6",
     "copy-webpack-plugin": "^4.6.0",
     "cross-env": "^5.2.0",
     "css-loader": "^1.0.1",
+    "cssimportant-loader": "^0.4.0",
     "eslint": "^5.3.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.0",
     "node-sass": "^4.10.0",
     "node.normalize.scss": "^3.0.3",
+    "postcss-loader": "^3.0.0",
     "ramda": "^0.26.0",
     "react": "^16.6.1",
     "react-dom": "^16.6.1",
+    "react-draggable": "^3.0.5",
     "react-redux": "^5.1.1",
     "redux": "^4.0.1",
-    "react-draggable": "^3.0.5",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
-    "postcss-loader": "^3.0.0",
     "svg-inline-loader": "^0.8.0",
     "svg-inline-react": "^3.1.0",
     "val-loader": "^1.1.1",
     "webpack": "^4.25.1",
     "webpack-cli": "^3.1.2",
     "whatwg-fetch": "^3.0.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "moment": "^2.23.0",
+    "moment-timezone": "^0.5.23",
+    "react-moment": "^0.8.4",
+    "react-shadow-dom-retarget-events": "^1.0.8",
+    "url-loader": "^1.1.2"
+  },
```

Comment:
^ Other frontend dependencies have been dev dependencies, doesn't really matter I think except for consistency?